enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "0.4.1"
+__version__ = "0.5.0"
```

Comment:
`0.5.0` is the current version in [edx-enterprise repo](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py#L7), any change expected here ?